### PR TITLE
Add "View" menu and update shortcuts

### PR DESCRIPTION
**Pull request by reichlfl**
Adding the new "Track Configuration Pane" to the main menu as well, and restructuring the menu and shortcuts a bit:

- "Source Path Mappings" now show up in the main menu, but no longer under "Capture Options"
- "Capture Options" and "Source Path Mappings" are both under a new menu entry "Settings"
- Introduce a new "View" menu that contains view & windows - related entries (toggling help, toggling the track configuration pane, toggling introspection)

The "Help" button was changed into a toggle, so its state is consistent with the display of the help overlay in `CaptureWindow`. To keep this consistency, `CaptureWindow` is no longer handling the `H` hotkey, but instead, this is now handled by a global Qt Keyboard Shortcut `Ctrl + H`.

For consistency, start / stop capture was also moved to a Qt Keyboard Shortcut `F5`. The old hotkey `X` is still working, but not displayed anywhere. Introspection still works as before. `F5` will now work globally, so a capture can also be restarted when e.g. browsing the sampling table.

**UPDATE** I've changed the shortcut (used to be `Ctrl + X`) to `F5` to not conflict with the usual `Cut` shortcut.

### CODE REVIEW

File: src/OrbitQt/orbitmainwindow.ui [hunk 1]
```diff
@@ -487,17 +496,20 @@ QPushButton:disabled {
    <property name="toolTip">
     <string>Toggle Capture</string>
```
Comment (antonrohr) [inline src/OrbitQt/orbitmainwindow.ui:497]: As an idea: Maybe we should also indicate the shortcuts in these tool tips here. 
"Toggle Capture (Ctrl+ X)" 
(Same for help)

**Review by antonrohr — CHANGES_REQUESTED** [review]
I think this is a good change and helps decluttering the UI!

Regarding the track configuration pane: Is this supposed to show the tracks hiding yet? In my test it is just empty. I would prefer to hide it behind a flag for now. Or are you confident the rest of the functionality is making it in before the branch cut? I am setting this to review changes to clarify that; everything else LGTM (just nits)

**Comment by reichlfl** [comment]
> I think this is a good change and helps decluttering the UI!
> 
> Regarding the track configuration pane: Is this supposed to show the tracks hiding yet? In my test it is just empty. I would prefer to hide it behind a flag for now. Or are you confident the rest of the functionality is making it in before the branch cut? I am setting this to review changes to clarify that; everything else LGTM (just nits)

Oh, sorry, this is actually supposed to be behind a flag (-enable_track_type_visibility_feature), I forgot to hide the menu entry in addition to the button. I've fixed this.

File: src/OrbitQt/orbitmainwindow.cpp [hunk 2]
```diff
@@ -1082,7 +1082,12 @@ void OrbitMainWindow::on_actionCaptureOptions_triggered() {
   LoadCaptureOptionsIntoApp();
 }
 
-void OrbitMainWindow::on_actionHelp_triggered() { app_->ToggleDrawHelp(); }
+void OrbitMainWindow::on_actionHelp_toggled(bool checked) {
+  CaptureWindow* capture_window = dynamic_cast<CaptureWindow*>(ui->CaptureGLWidget->GetCanvas());
```
[the remark this answers is not in the document]
Comment (reichlfl) [reply src/OrbitQt/orbitmainwindow.cpp]: +1, changed

File: src/OrbitQt/orbitmainwindow.cpp [hunk 3]
```diff
@@ -1082,7 +1082,12 @@ void OrbitMainWindow::on_actionCaptureOptions_triggered() {
...
+  if (capture_window != nullptr) {
```
[the remark this answers is not in the document]
Comment (reichlfl) [reply src/OrbitQt/orbitmainwindow.cpp]: Not really, so I agree, a `CHECK` is better. I was mostly thinking of a failed `dynamic_cast`, but this should then actually be an explicit error and not a silent failure.

File: src/OrbitQt/orbitmainwindow.ui (hunk 1 above)
Comment (reichlfl) [reply src/OrbitQt/orbitmainwindow.ui]: Good idea, I thought QT would display them automatically, but apparently not. I've added them, thanks.

Comment (antonrohr) [reply src/OrbitQt/orbitmainwindow.ui]: They are added in the file menu automatically, but apparently not in the tooltips.

**Review by antonrohr — APPROVED** [review]
Thanks for incorporating the changes! I tested again with `--enable_track_type_visibility_feature` and it all looks/works great.

**Pull request merged**